src/main/java/seedu/exercise/logic/commands/DeleteRegimeCommand.java

```diff
@@ -25,13 +25,14 @@
 import seedu.exercise.model.resource.Regime;
 import seedu.exercise.ui.ListResourceType;
 
+//@@author jietung
 /**
  * Deletes a regime identified using it's name or deletes exercises in regime.
  */
 public class DeleteRegimeCommand extends DeleteCommand implements PayloadCarrierCommand {
 
-    public static final String MESSAGE_USAGE_REGIME = "Parameters: REGIME NAME, INDEX (must be a positive integer)\n"
-            + "\t\tExample: "
+    public static final String MESSAGE_USAGE_REGIME = "Parameters: REGIME NAME, INDEX (must be a positive integer)"
```

Comment:
most of us have placed the `PREFIX_NAME` in front of the message usage. Please keep it standard

Reply:
ok